plugins/jira-dashboard/src/components/JiraTable/cells/AssigneeCell.tsx

```diff
@@ -0,0 +1,61 @@
+import { Avatar, Link } from '@backstage/core-components';
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+import { Issue } from '@axis-backstage/plugin-jira-dashboard-common';
+import { EntityPeekAheadPopover } from '@backstage/plugin-catalog-react';
+import { stringifyEntityRef } from '@backstage/catalog-model';
+
+type Props = {
+  assignee?: Issue['fields']['assignee'];
+};
+
+export const AssigneeCell = ({ assignee }: Props) => {
+  if (!assignee) {
+    return null;
+  }
+
+  if (!assignee.name || assignee.name.toLowerCase() === 'unassigned') {
+    return (
+      <Stack
+        direction="row"
+        gap={1}
+        alignItems="center"
+        data-testid="assignee-avatar"
+      >
+        <Avatar picture="" customStyles={{ width: 35, height: 35 }} />
+        <Typography variant="body2">Unassigned</Typography>
+      </Stack>
+    );
+  }
+
+  const entityRef = {
+    kind: 'user',
+    namespace: 'default',
+    name: assignee.name,
+  };
+
+  return (
+    <EntityPeekAheadPopover entityRef={stringifyEntityRef(entityRef)}>
+      <Link
+        to={`/catalog/${entityRef.namespace}/${
+          entityRef.kind
+        }/${stringifyEntityRef(entityRef).split('/').pop()}`}
+      >
+        <Stack
+          direction="row"
+          gap={1}
+          alignItems="center"
+          data-testid="assignee-avatar"
+        >
+          <Avatar
+            picture={assignee.avatarUrls?.['48x48'] || ''}
+            customStyles={{ width: 35, height: 35 }}
+          />
+          <Typography noWrap>
```

Comment:
Let's add a variant='body2' here so the font size is the same as the rest of the table?